src/main/java/com/junichi11/netbeans/modules/backlog/issue/ui/BacklogIssuePanel.java

```diff
@@ -1445,22 +1450,15 @@ public void run() {
         "BacklogIssuePanel.message.uploading.attachments=Uploading files"
     })
     private void selectFiles() {
-        // attachment
-        RP.post(new Runnable() {
 
+        class AttachmentUploader implements Runnable {
+            
+            File[] attachments;
+            
             @Override
             public void run() {
+     
                 submitHeaderButton.setEnabled(false);
-                // TODO add FileFilter
```

Comment:
Now I'm quite embarrassed, at start, before I decided to try create PRs, I was just messing with the code trying to run the plugin in my netbeans, and removed it accidentally. I'm sorry need to check the diffs better.